caliban_toolbox/utils/io_utils.py

```diff
@@ -122,7 +122,7 @@ def save_npzs_for_caliban(X_data, y_data, original_data, log_data, save_dir,
                 raise NotImplementedError()
 
     log_data['fov_names'] = fov_names.tolist()
-    log_data['channel_names'] = original_data.channels.values.tolist()
+    log_data['label_name'] = str(y_data.coords['compartments'][0].values)
```

Comment:
Could this be generalized to not require the string `"compartments"`?

Reply:
is it possible to get the `"compartments"` from `y_data` itself? Like `list(y_data.ndim)[-1]` or something? I believe I saw that pattern (or something similar) used elsewhere in this PR.

Reply:
Yeah, it just looks a little messier. Would that be better than having it as a global?

Reply:
Yeah I think it would be a little more flexible than the global var.

Reply:
I realize this is annotations rather than raw data, but how does this nomenclature work with phase images/channels?

Reply:
'Compartments' is the name of the axis that holds labeled data. Within that axis, you can specify what the compartment is: nuclear, whole-cell, cytoplasm, etc. What this does is handle any of those potential names by getting the label you have provided within the compartment axis.